packages/configuration/src/get-server-configuration.ts

```
@@ -0,0 +1,59 @@
+import { SERVER_CONFIGURATION_KEYS } from './types.js';
+
+function getServerConfigurationValue<T>(
+  configuration: StoredConfiguration | null,
+  key: SERVER_CONFIGURATION_KEYS,
+  defaultValue: T
+): T {
+  if (configuration?.values[key] === undefined) {
+    return defaultValue;
+  }
+
+function getServerConfigurationValue<T>(
```

Biome (1.9.4)

[error] This function is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)